test/html_test.js

```diff
@@ -68,5 +69,54 @@ exports.htmllint = {
         file: path.join('test', 'invalid.html')
       }
     ], 'one error from test/invalid.html, other three were ignored');
+  },
+  'partialWrapperTemplatePath': function(test) {
```

Comment:
This test covers a lot of the partial implementation details, but doesn't actually test if the wrapped file passes validation. Would be good to add that.